app/form_models/jobseekers/job_application/personal_details_form.rb

```diff
@@ -1,7 +1,21 @@
 class Jobseekers::JobApplication::PersonalDetailsForm
   include ActiveModel::Model
 
-  attr_accessor :first_name
+  attr_accessor :building_and_street, :email_address, :first_name, :last_name, :national_insurance_number,
+                :phone_number, :previous_names, :postcode, :teacher_reference_number, :town_or_city
 
-  validates :first_name, presence: true
+  EMAIL_FORMAT = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/.freeze
+  PHONE_NUMBER_FORMAT = Devise.email_regexp
```

Comment:
wrong place, this is for the email not the phone number

Reply:
Bloody hell. Apologies.